express/scripts/utils/free-plan.js

```diff
@@ -0,0 +1,92 @@
+import {
+  getLibs,
+  getLottie,
+  lazyLoadLottiePlayer, 
+} from '../utils.js';
+
+import { fetchPlaceholders } from './fetch-placeholders.js';
```

Comment:
as said, use the placeholders approach from milo. We have examples of how to do this.